web/portal/src/main/resources/locale/portal/webui_en.properties

```diff
@@ -1338,11 +1338,11 @@ gatein.forgotPassword.captchaError=Captcha Verification Failed
 ###### Onboarding feature
 onboarding.email.subject=Invitation to join the platform
 onboarding.email.header=Invitation to join the platform
-onboarding.email.hello=Hello
+onboarding.email.hi=Hi
```

Comment:
We will keep the same labels right now, no modification needed

Reply:
Done!